tensorflow/compiler/mlir/tensorflow/ir/tf_ops.cc

```diff
@@ -2404,8 +2406,9 @@ static LogicalResult Verify(OneHotOp op) {
   int64_t axis = op.axis().getSExtValue();
 
   auto indices_ty = op.indices().getType().dyn_cast<RankedTensorType>();
+  const int64 indices_ty_getShape_size = indices_ty.getShape().size();
```

Comment:
Variable does not follow coding style regarding naming of local variables: the name should be in `snake_case`.